common/compositor/va/varenderer.cpp

```diff
@@ -42,11 +43,27 @@ VARenderer::~VARenderer() {
   }
 }
 
+static uint32_t GetScalingModeFromEnv() {
+  char value[PROPERTY_VALUE_MAX];
+  property_get("board.hwc.scaling.mode", value, "0");
```

Comment:
This cannot go here but in iahwc side.

Reply:
@kalyankondapally , If we put this in iahwc class, it will need tons of api to pass only this one flag from iahwc to varender which need to much effort. So I think it's better to keep this here. How do you think of this?